src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -89,10 +89,11 @@ RequestRateManager::InitManagerFinalize()
 }
 
 cb::Error
-RequestRateManager::ChangeRequestRate(const double request_rate)
+RequestRateManager::ChangeRequestRate(
+    const double request_rate, const size_t request_count)
 {
   PauseWorkers();
-  ConfigureThreads();
+  ConfigureThreads(request_count);
```

Comment:
How does this feature work if someone asks for a range of request rates or concurrencies?

Reply:
Good catch. It does not work, as the threads have exited. I added a check stating that sweeping over rr or conc is not allowed with request-count